modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/SeriesEndpoint.java

```diff
@@ -516,8 +516,11 @@ public Response getNewThemes() {
 
     JSONObject themesJson = new JSONObject();
     for (SearchResultItem<Theme> item : results.getItems()) {
+      JSONObject list = new JSONObject();
```

Comment:
`list` is not the most descriptive variable name, especially since this is an object, not a list. How about `theme` (or `themeJson` for consistency …)